minimal.js

```diff
@@ -93,9 +93,24 @@ module.exports = function (dirname, keys, opts) {
 
   var log = OffsetLog(path.join(dirname, 'log.offset'), {blockSize:1024*16, codec:codec})
 
-  //NOTE: must use db.ready.set(true) at when migration is complete
+  const maps = []
+  const chainMaps = (val, cb) => {
+    if (!maps.length) return cb(null, val)
```

Comment:
I was thinking I'd be able to squeeze out a few milliseconds of performance by checking a boolean here instead, but [my benchmarks](https://jsperf.com/arrboolcheck) are showing that `if (!maps.length)` is just as fast as checking `if (hasMaps)` or similar.